app/controllers/api/v1/posts.rb

```diff
@@ -0,0 +1,20 @@
+module API
+  module V1
+    class Posts < Grape::API
+      include API::V1::Defaults
+      resource :posts do
+        desc 'Return all posts'
+        get '' do
+          Post.all
+        end
+        desc 'Return a post'
+        params do
+          requires :id, type: String, desc: 'ID of the post'
+        end
+        get ':id' do
+          Post.where(id: permitted_params[:id]).first!
+        end
```

Comment:
- [ ] You should have authentication for API endpoints. 
If I am not logged in user I should not be able to see or create a post or comment